deep_speech_2/test_ctc_beam_search_decoder.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import array_ops
```

Comment:
It is not proper to include tensor-flow dependency. It would be better to paste ground-truth results and just compare our results with it.